sci-libs/composable-kernel/files/composable-kernel-5.7.1-enable-test-examples.patch

```diff
@@ -0,0 +1,19 @@
+--- a/CMakeLists.txt
```

Comment:
Not a blocker for this specific PR but please send these patches upstream & include a link to the PR/commit at the top of the patch.